docs/source/datasets/index.rst

```diff
@@ -42,80 +42,123 @@ The following simulant attributes are included in this dataset:
    :header-rows: 1
 
    * - Attribute Name
-     - Column Name    
+     - Column Name
+     - Notes    
    * - Unique simulant ID
      - :code:`simulant_id`
+     - Not affected by noise functions; intended use is 'ground truth' for PRL tracking. 
    * - First name
      - :code:`first_name`
+     - 
    * - Middle initial
      - :code:`middle_initial`
+     - 
    * - Last name
      - :code:`last_name`
+     - 
    * - Age
-     - :code:`age`  
+     - :code:`age` 
+     - Rounded to nearest integer. 
    * - Date of birth
      - :code:`date_of_birth`
+     - Formatted as MM/DD/YYYY.
```

Comment:
I'm not sure if the engineers will have time for this, please double check if it is in this format before documenting it.